src/Input/InputEvents.cpp

```diff
@@ -479,3 +491,17 @@ InputEvents::ProcessTimer()
   DoQueuedEvents();
   ProcessMenuTimer();
 }
+
+void  //////// LOCKSCREEN
+InputEvents::eventLockscreen(const TCHAR *mode)
+{
+  LockScreen = !LockScreen;
```

Comment:
You declared `LockScreen` as unsigned integer, but not you're using it as boolean. Now what is it really? And what purpose does it serve?